jobrunner/secrets-template.env

```diff
@@ -0,0 +1,22 @@
+# These are secrets that should not be kept privately
+
+# Credentials for logging into the job server
+JOB_SERVER_TOKEN=
+
+# A Github developer token that has read access to private repos
+PRIVATE_REPO_ACCESS_TOKEN=
+
+# A database containing dummy data; results from this could be freely
+# published without review
+DUMMY_DATABASE_URL=
+
+# A database containing a slice of the full data; useful for checking
+# or debuggin real data without potentially having to wait for hours
+# for completion
+SLICE_DATABASE_URL=
+
+# The full database
+FULL_DATABASE_URL=
+
+# License for running stata images
```

Comment:
Is this a path to a license or a license key?

Reply:
The key itself as a string - will document